data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/PeerForwarderConfiguration.java

```diff
@@ -87,7 +89,9 @@ public PeerForwarderConfiguration (
             @JsonProperty("batch_size") final Integer batchSize,
             @JsonProperty("batch_delay") final Integer batchDelay,
             @JsonProperty("buffer_size") final Integer bufferSize,
-            @JsonProperty("drain_timeout") final Duration drainTimeout
+            @JsonProperty("drain_timeout") final Duration drainTimeout,
+            @JsonProperty("failed_forwarding_requests_local_write_timeout") final Integer failedForwardingRequestLocalWriteTimeout,
+            @JsonProperty("forwarding_requests_workers") final Integer forwardingRequestWorkers
```

Comment:
This option is similar to `client_thread_count` which is deprecated with this change. Can you remove that as well?

Reply:
What was `client_thread_count` used for beforehand?

Reply:
It was used for the ExecutorService in the PeerForwarderClient: https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/client/PeerForwarderClient.java#L55

Reply:
OK. So it really is the same thing from what I can see.